### PR TITLE
[Remove Stream/Optional usage] /LPS-174240 Remove_usages_of_Stream_usages_for_portal-store-azure

### PR DESCRIPTION
**Pull request by jeffwu0724**
Hi Tina,

Resent: https://github.com/jeffwu0724/liferay-portal/pull/241
Use forEach instead of using for loop in this PR
This is for ticket https://issues.liferay.com/browse/LPS-174240.
Thanks for checking.

### CODE REVIEW

**Comment by jeffwu0724** [comment]
ci:test:dl-store

**Comment by zhangyan0701** [comment]
No regressions found.

File: modules/apps/portal-store/portal-store-azure/src/main/java/com/liferay/portal/store/azure/AzureStore.java [hunk 1]
```diff
@@ -256,21 +255,22 @@ public String[] getFileVersions(
...
-			}
-		).sorted(
-			DLUtil::compareVersions
-		).toArray(
-			String[]::new
-		);
+				fileVersions.add(blobItemName.substring(prefix.length()));
+			});
+
+		Collections.sort(fileVersions, DLUtil::compareVersions);
```
Comment (dantewang) [inline modules/apps/portal-store/portal-store-azure/src/main/java/com/liferay/portal/store/azure/AzureStore.java:271]: @jeffwu0724 Please directly use `List.sort` as IntelliJ IDEA recommends. You can search for the difference between these two.

File: modules/apps/portal-store/portal-store-azure/src/main/java/com/liferay/portal/store/azure/AzureStore.java [hunk 2]
```diff
@@ -208,14 +208,13 @@ public String[] getFileNames(
 		PagedIterable<BlobItem> pagedIterable = _blobContainerClient.listBlobs(
 			listBlobsOptions, null);
 
-		Stream<BlobItem> stream = pagedIterable.stream();
+		List<String> fileNames = new ArrayList<>();
```
Comment (dantewang) [inline modules/apps/portal-store/portal-store-azure/src/main/java/com/liferay/portal/store/azure/AzureStore.java:211]: @jeffwu0724 Please double check the SF in https://github.com/brianchandotcom/liferay-portal/pull/130838, if Brian accepts it, please apply the same pattern here.

**Review by dantewang — CHANGES_REQUESTED** [review]
@jeffwu0724 Please see my comment and resend, thanks.

**Pull request closed without merging**